types/setting.go

```diff
@@ -86,6 +86,7 @@ const (
 	SettingNameDefaultLonghornStaticStorageClass                        = SettingName("default-longhorn-static-storage-class")
 	SettingNameTaintToleration                                          = SettingName("taint-toleration")
 	SettingNameSystemManagedComponentsNodeSelector                      = SettingName("system-managed-components-node-selector")
+	SettingNameSystemManagedComponentsResourceLimits                    = SettingName("system-managed-components-resource-limits")
```

Comment:
To align the new settings introduced by https://github.com/longhorn/longhorn-manager/pull/4251/files#diff-59782221a3f8c591d66ab9f73614211f67aba44b4ad4a31329f875f3ff0eeb25

```suggestion
	SettingNameSystemManagedCSISidecarComponentsResourceLimits                    = SettingName("system-managed-csi-sidecar-components-resource-limits")
```